lib/rules/prefer-find-by.ts

```diff
@@ -166,17 +391,21 @@ export default createTestingLibraryRule<Options, MessageIds>({
           });
           return;
         }
-        if (
-          !ASTUtils.isIdentifier(argument.body.callee) ||
-          !helpers.isSyncQuery(argument.body.callee)
-        ) {
+
+        if (!isSyncQuery(argument)) {
           return;
         }
+
         // shape of () => getByText
```

Comment:
Does this comment make sense anymore? Maybe this one (and others across the file) must be updated.

Reply:
Well, it's still accurate in the sense that it covers `() => getByText`, but it also includes `getByText` wrapped in a presence matcher, so I am not sure if I should update the comment to reflect that